tests/FakeItEasy.Tests/FakeItEasy.Tests.csproj

```diff
@@ -273,4 +272,4 @@
     </PropertyGroup>
     <Error Condition="!Exists('..\..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets'))" />
   </Target>
-</Project>
+</Project>
```

Comment:
Don't worry about blocking my PR. As always, I'm happy to make any change, no matter how small, if it makes sense.
But if we have to manually revert these changes every time we have VS edit a .csproj, I'm with @thomaslevesque—I think we should go with its flow.